src/riscv32_csr_m.c

```diff
@@ -17,30 +17,51 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 */
 
+#include "bit_ops.h"
 #include "riscv32.h"
 #include "riscv32_csr.h"
 
 #define CSR_MARCHID 0x5256564D // 'RVVM'
 
-// no N extension, U_x bits are hardwired to 0
-#define CSR_MSTATUS_MASK 0x807FF9CC
 #define CSR_MEIP_MASK    0xAAA
 
 #define CSR_MISA_RV32  0x40000000
 #define CSR_MISA_RV64  0x80000000
 #define CSR_MISA_RV128 0xC0000000
```

Comment:
Needs to be shifted for RV64 as well

Reply:
It is, that's a removal. This constant is now replaced with csr_mstatus_mask function